packages/core/src/org/SFPOrg.ts

```diff
@@ -13,7 +13,7 @@ export default class SFPOrg extends Org {
     public async getInstalledArtifacts() {
         try {
             let records = await QueryHelper.query<SfpowerscriptsArtifact2__c>(
-                'SELECT Id, Name, CommitId__c, Version__c, Tag__c FROM SfpowerscriptsArtifact2__c',
+                'SELECT Id, Name, CommitId__c, Version__c, Tag__c FROM SfpowerscriptsArtifact2__c ORDER BY CommitId__c ASC',
```

Comment:
What's the reasoning for ordering by commit id?

Reply:
Reducing the number of checkout operations